composer.json

```diff
@@ -10,12 +10,16 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=7.2.0",
```

Comment:
This kind of change feels like a major version bump to 3 is warranted here, not a minor 2.3 release.

Reply:
Yeah, I didn't think this versioning through all that well. PSR-14 implementations require PHP >=7.2 so I just bumped it to fit that requirement. I'll revise `changelog.md` to reflect this, thanks!